### PR TITLE
install sriov-infiniband-cni along with sriov-cni

### PR DESCRIPTION
**Pull request by zshi-redhat**

### CODE REVIEW

**Comment by zshi-redhat** [comment]
/hold
hold until the image is built by ART.

File: bindata/manifests/plugins/sriov-cni.yaml [hunk 1]
```diff
@@ -27,6 +27,12 @@ spec:
       tolerations:
       - operator: Exists
       serviceAccountName: sriov-cni
+      initContainers:
```
Comment (pliurh) [inline bindata/manifests/plugins/sriov-cni.yaml:30]: Why using initContainter for ib-cni? Cann't we use the same approach as the sriov-cni?

Comment (zshi-redhat) [reply bindata/manifests/plugins/sriov-cni.yaml:30]: To save system resource, With initContainer, we just use one daemonset for all CNI binary copy, this is the same as how Multus related CNI binaries are copied in cluster network operator.

**Pull request merged**